videospace_pot/server/__init__.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
```

Comment:
Do you want me to remove copyright Kitware from all files which are copied from imagespace?